origin-dapp/src/components/compact-messages.js

```diff
@@ -101,6 +101,7 @@ export default class CompactMessages extends Component {
           message={message}
           contentOnly={contentOnly}
           previousOfferMessage={previousOfferMessage}
+          previousMessage={isFirstMessage ? {} : sortedMessages[i-1]}
```

Comment:
Did you mean to add this in the old dapp?

Reply:
@nick 🤦‍♀️